backend/dal/Services/PersonService.cs

```diff
@@ -66,17 +67,37 @@ public PimsPerson Get(long id)
         /// <summary>
         /// Add the specified person contact to the datasource.
         /// </summary>
-        /// <param name="add"></param>
+        /// <param name="person"></param>
+        /// <param name="userOverride"></param>
         /// <returns></returns>
-        public PimsPerson Add(PimsPerson add)
+        public PimsPerson Add(PimsPerson person, bool userOverride)
         {
-            add.ThrowIfNull(nameof(add));
-            this.Context.PimsPeople.Add(add);
+            person.ThrowIfNull(nameof(person));
+            if (!userOverride)
+            {
+
+                List<PimsPerson> existingPersons = this.Context.PimsPeople.Where(
+                    p => EF.Functions.Collate(p.FirstName, "SQL_Latin1_General_CP1_CI_AS") == person.FirstName &&
```

Comment:
suggestion: could we have the collection ID put in a constant/enum so we don't have to copy/paste every time (with potential of typos etc_?

Reply:
sure can!